odl/solvers/advanced/chambolle_pock.py

```diff
@@ -108,6 +105,39 @@ def chambolle_pock_solver(op, x, tau, sigma, proximal_primal, proximal_dual,
 
     Notes
     -----
+    The problem of interest is:
+
+    .. math::
+
+        \\min_{x \\in X} F(K x) + G(x)
+
+    Where the technical conditions are that :math:`K` is an operator
+    between hilbert spaces :math:`X` and :math:`Y`, where convergence is only
+    guaranteed if :math:`K` is linear and :math:`X, Y` are finite dimensional.
+    Further, :math:`G : X -> [0, +\\infty]` and :math:`F : Y -> [0, +\\infty]`
+    are proper, convex, lower-semicontinuous functionals.
+
+    It is often of interest to study problems that involve several operators,
+    for example the classical TV regularized problem
+
+    .. math::
+
+        \\min_x ||Ax - b||_2^2 + ||\\nabla x||_1
+
+    Here it is often tempting to let :math:`K=A`, :math:`F(y)=||y||_2^2` and
+    :math:`G(x)=||\\nabla x||_1`. This is however not feasible, since the
+    proximal of :math:`||\\nabla .||_1` has no closed form expression.
+
+    Instead, the problem can be formulated :math:`K(x) -> (A(x), \\nabla x)`,
+    :math:`F((x_1, x_2)) -> ||x_1||_2^2 + ||x_2||_1`, :math:`G(x)=0`. See the
+    examples folder for more information on how to do this.
```

Comment:
Will fix all except `\\lVert`, mostly because it adds quite little and is impossible to read for someone without some latex experiance (`\\to` by comparison is quite obvious).